config/settings_prod.py

```diff
@@ -132,6 +132,24 @@
 LOG_FILE_SIZE = 1024 * 1024 * 10  # 10mb
 LOG_FILE_COUNT = 3
 
+# Global API throttle rates. Note that these can be customized per view if needed.
+ANON_API_THROTTLE_RATE = get_env("ANON_API_THROTTLE_RATE", default="10/minute")
+USER_API_THROTTLE_RATE = get_env("USER_API_THROTTLE_RATE", default="100/minute")
```

Comment:
These rates are up to discussion.
I chose what I believe are sane values as I don't see why a user would need to make more than 100 requests per minute. If there is some automated process that requires more, let me know, although if that is the case maybe the process needs to be redesigned so that we don't hit the API that much (e.g. bulk operations).

Reply:
@dmonzonis I am thinking about VM team scripts. I haven't seen all of them but minimally one of them is creating kernel Flaws and adding like 10 affects for each of those Flaws, that might potentially exceed that limit. This can be mitigated by using a bulk actions however they are not yet exposed in the osidb-bindings, I would need to prioritize that work.

Reply:
I agree with @JakubFrejlach. I think one user could generate up to 400 requests per minute using this approach and lately maybe even more because it has been optimized I think. Given that there are flaws with 600 affects, I propose to set this to 600/minute and lower it later when the dust settles.

Reply:
Is the issue only with affects? I could set a higher limit for the affect API endpoint for the time being, instead of setting it globally.

Reply:
The bindings could ideally handle that and lower the load but it is some additional work of course

Reply:
Probably yes, I'm aware only of affects needing this. Mass tracker filing is probably slow enough not to hit the limit.